data/regions/IE.yml

```diff
@@ -13,7 +13,6 @@ tags:
 zip_regex: "^(D6W|[AC-FHKNPRT-Z]\\d{2}) ?[0-9AC-FHKNPRT-Z]{4}$"
 partial_zip_regex: "^(D6W|[AC-FHKNPRT-Z]\\d{2})$"
 zip_example: A65 F4E2
-zip_requirement: recommended
```

Comment:
As of https://github.com/Shopify/worldwide/pull/320, by removing this field, a country with a zip_regex will lead to `zip_required?` => `true`.